src/blockdata/transaction.rs

```diff
@@ -426,6 +426,13 @@ impl Transaction {
         self.get_scaled_size(1)
     }
 
+    /// Gets the "vsize" of this transaction. Will be `ceil(weight / 4.0)`.
+    #[inline]
+    pub fn get_vsize(&self) -> usize {
```

Comment:
Why do we need to expose this? Users can just divide-rounding-up themselves, and I'm not sure how much users should/do have a use for this?

Reply:
`tx.get_vsize()` is readable and handy. Any reason for the user not to use such this instead of comming up with their own version?